### DISCUSSION TITLE
Can we make two simulation runs using one Andes engine

### DISCUSSION THREAD

**Discussion by vaahedi** [CURENT/andes#506] [Ideas]
Hello everyone,

Can we take advantage of Andes engine and execute two different or n different usecases

**Comment by jinningwang** [comment]
The documentation has discussed batch simulation. Would you mind giving more details?

**Reply by vaahedi** [reply]
Hello Jinning,
I am asking if we can use one Andes engine on a server to respond to two users in parallel executing different cases simultaneously

**Reply by jinningwang** [reply]
This example discussed in-memory batch simulation, https://docs.andes.app/en/latest/_examples/ex7.html, although I think cases are executed one by one.
Parallel simulation might be achieved with other Python tools, such as multiprocessing.

Is there any specific needs for parallel simulation? It can be difficult to interacte with paralleled process before they are finished.

**Reply by vaahedi** [reply]
Thank you Jinning, the in-memory example you provided is what I was looking for. Thank you
